### PR TITLE
Bump mem/cpu for the local-e2e-containerized jobs

### PR DESCRIPTION
**Pull request by dims**
See if bumping CPU/Mem can avoid the timeouts we have been seeing in these jobs.

Change-Id: I224d8e7017d35d101e21f7249430c942f5e2501d

### CODE REVIEW

**Comment by dims** [comment]
/assign @BenTheElder 

Ben, i copied what you had for `kind` ci jobs.

File: config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml [hunk 1]
```diff
@@ -64,3 +71,10 @@ periodics:
       # docker-in-docker needs privileged mode
       securityContext:
         privileged: true
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml]: running this on every PR for a long time might (?) be an issue. kind is only running a couple of CI jobs for 1hr plus, and some < 20m presubmits on a small repo (kind).

**Comment by BenTheElder** [comment]
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/71679/pull-kubernetes-local-e2e-containerized/3766/artifacts/kubetest-local657713592/kubelet.log

note that the kubelet is seeing the whole host node memory and complaining that it ran out:

```
I1205 01:43:45.225893  129008 helpers.go:812] eviction manager: observations: signal=nodefs.available, available: 133122876Ki, capacity: 253696108Ki, time: 2018-12-05 01:43:45.211705743 +0000 UTC m=+5987.960653863
I1205 01:43:45.225960  129008 helpers.go:812] eviction manager: observations: signal=nodefs.inodesFree, available: 14027091, capacity: 16050720, time: 2018-12-05 01:43:45.211705743 +0000 UTC m=+5987.960653863
I1205 01:43:45.225971  129008 helpers.go:812] eviction manager: observations: signal=imagefs.available, available: 133122876Ki, capacity: 253696108Ki, time: 2018-12-05 01:43:45.211705743 +0000 UTC m=+5987.960653863
I1205 01:43:45.225977  129008 helpers.go:812] eviction manager: observations: signal=imagefs.inodesFree, available: 14027091, capacity: 16050720, time: 2018-12-05 01:43:45.211705743 +0000 UTC m=+5987.960653863
I1205 01:43:45.225983  129008 helpers.go:812] eviction manager: observations: signal=pid.available, available: 4192749, capacity: 4Mi, time: 2018-12-05 01:43:45.223832867 +0000 UTC m=+5987.972780975
I1205 01:43:45.225990  129008 helpers.go:812] eviction manager: observations: signal=memory.available, available: 46779652Ki, capacity: 53589632Ki, time: 2018-12-05 01:43:45.211705743 +0000 UTC m=+5987.960653863
I1205 01:43:45.225996  129008 helpers.go:812] eviction manager: observations: signal=allocatableMemory.available, available: 53532508Ki, capacity: 53589632Ki, time: 2018-12-05 01:43:45.225772274 +0000 UTC m=+5987.974720394
I1205 01:43:45.226030  129008 eviction_manager.go:317] eviction manager: no resources are starved
```

53532508Ki

**Comment by dims** [comment]
@BenTheElder so drop the memory `"9000Mi"` change?

File: config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml]: sorry this dropped off my radar, I'm not really sure how to fix this issue.

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml]: I don't think we have support for this.

**Comment by BenTheElder** [comment]
/approve
/hold
genuinely not sure what the best option is here right now.

**Comment by dims** [comment]
@BenTheElder let's run it for a week and revisit?

/hold cancel

**Comment by yujuhong** [comment]
The job is building and running tests with `--ginkgo-parallel=1`, do we really believe it should complete in 

> [](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/71679/pull-kubernetes-local-e2e-containerized/3766/artifacts/kubetest-local657713592/kubelet.log)
> 
> note that the kubelet is seeing the whole host node memory and complaining that it ran out:

Where did kubelet say that it was out of memory? The log you pasted had `no resources are starved`.

**Comment by dims** [comment]
Ack, i'll remove the `==1`

**Comment by yujuhong** [comment]
> Ack, i'll remove the `=1`

I think the tests are running on one node(?). Don't think we want to make it too parallel... (btw, I don't know much about this job, so feel free to ignore me)

**Comment by dims** [comment]
@yujuhong am ok to give it a try, let's see if it stabilizes things, the cpu/mem was also an experiment anyway ..

**Comment by BenTheElder** [comment]
er sorry that part doesn't show eviction, but if you look at the capacity that's clearly the host's capacity

File: config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ presubmits:
         - --
         - --build=bazel
         - --deployment=local
-        - --ginkgo-parallel=1
+        - --ginkgo-parallel
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml]: this definitely makes things slightly flakier for `kind` at least.
https://testgrid.k8s.io/conformance-kind
https://testgrid.k8s.io/conformance-kind#kind,%20master%20(dev)%20%5Bnon-serial%5D

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml]: I plan to experiment with values for this at some point to see if reducing it helps, otoh to me this signals that our tests are bad, I'm pretty certain that we have some conformance tests using events which I thought was a no-no ...

**Comment by dims** [comment]
@yujuhong please take a look again (happy new year!) i'll watch the job carefully and get it back on its feet.

**Comment by dims** [comment]
@BenTheElder please lgtm if appropriate

**Comment by BenTheElder** [comment]
sorry! thought this was already in 🙄

**Comment by BenTheElder** [comment]
/lgtm
Someday this will merge 🙃

**Pull request merged**